libevmcore/Instruction.cpp

```diff
@@ -82,6 +82,8 @@ const std::map<std::string, Instruction> dev::eth::c_instructions =
 	{ "SSTORE", Instruction::SSTORE },
 	{ "JUMP", Instruction::JUMP },
 	{ "JUMPI", Instruction::JUMPI },
+	{ "JUMPV", Instruction::JUMP },
```

Comment:
I thought so too, but they need to be used like regular opcodes, so they need to be in the same tables.  If ever new opcodes conflict there will be plenty of time to reassign these internally.  We could make an EIP issue to reserve a range of opcodes for this purpose, but that could delay this thing by months.  Currently these are behind a config macro that is off.